napari/components/add_layers_mixin.py

```diff
@@ -394,6 +394,8 @@ def add_labels(
         An image-like layer where every pixel contains an integer ID
         corresponding to the region it belongs to.
 
+        Modifies the input array in-place.
```

Comment:
This isn't necessarily true, if one is only looking at the layer. I would say "Using the viewer's label editing tools (painting, erasing) will modify the input-array in-place. To avoid this, pass a copy as follows:"

You might also specify that you can *get* the painted labels back with:

```python
layer = viewer.add_labels(data.copy())
# do some painting/editing
result = layer.data`. 
```